site/content/troubleshooting-guide.md

```diff
@@ -1 +1,122 @@
-### TODO
\ No newline at end of file
+
+# Troubleshooting Guide
+
+This guide outlines some failure scenarios and their possible solutions.
+
+## Node.js not installed
+AWS.Deploy.Tools relies on AWS CDK to provision resources for the user's cloud application. AWS CDK requires Node.js to be installed in the user's machine.
+
+**Minimum Node.js version >= 10.13.0**
+
+See [here](https://nodejs.org/en/download/) to install Node.js on your system.
+
+## Docker related issues
+AWS.Deploy.Tool requires Docker to perform containerized deployments.
+
+The following recipes are container based:
```

Comment:
As mention in later comment on not sure having this list adds any value and will cause us a headache making sure it is up to date.

Reply:
Removed the list